demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -39,6 +41,25 @@ protected void onCreate(Bundle savedInstanceState) {
         textViewBatch1 = findViewById(R.id.batch_1);
         textViewBatch2 = findViewById(R.id.batch_2);
 
+        View buttonMigrate = findViewById(R.id.button_migrate);
+        buttonMigrate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                new Thread(new Runnable() {
+                    @Override
+                    public void run() {
+                        Migrator migrator = MigrationFactory.createVersionOneToVersionTwoMigrator(getApplicationContext(), getDatabasePath("downloads.db"), new Migrator.Callback() {
+                            @Override
+                            public void onMigrationComplete() {
+                                deleteDatabase("downloads.db");
+                            }
+                        });
+                        migrator.migrate();
+                    }
+                }).start();
+            }
+        });
```

Comment:
I guess we're not too fussed about this code since it's just here to exercise the lib?

Reply:
Precisely 👍